packages/components/src/components/Alert/Alert.tsx

```diff
@@ -229,31 +252,32 @@ export const Alert: FC<AlertProps> = ({
       <View style={{ flexDirection: 'row' }}>
         {iconDisplay}
         <View style={{ flex: 1 }}>
-          {header ? (
-            <View
-              // ref={viewRef}
-              accessible={true}
-              aria-label={headerA11yLabel || header}
-              role="heading">
-              <Text style={headerFont}>{header}</Text>
-            </View>
-          ) : null}
-          {header && (description || children) ? <Spacer /> : null}
-          {description ? (
-            <View
-              // ref={!header ? viewRef : undefined}
-              accessible={true}
-              aria-label={descriptionA11yLabel || description}>
-              <Text style={descriptionFont}>{description}</Text>
+          {_header()}
+          {expanded && (
+            <View style={{ flexDirection: 'row' }}>
+              <View style={{ flex: 1 }}>
+                {header && (description || children) ? <Spacer /> : null}
+                {description ? (
+                  <View
+                    accessible={true}
+                    aria-label={descriptionA11yLabel || description}>
+                    <Text style={descriptionFont}>{description}</Text>
+                  </View>
+                ) : null}
+                {description && children ? <Spacer /> : null}
+                {children}
+              </View>
+              {expandable && <Spacer horizontal size={36} />}
```

Comment:
If retaining doing it this way, shouldn't the Spacer be 26? 

10 Spacer + 16 wide chevron and then needing to scale 26->34 with font scaling.

Reply:
Updated to 26. Are you saying I need to add logic to set a max width for scaling? I suppose another approach could be to add another invisible Icon with the same props as the chevron, but seems a little hacky.

Reply:
> Are you saying I need to add logic to set a max width for scaling?

If we want to mimic what the chevron is doing when font scaling comes into play, we'd need to match it growing but only a certain amount otherwise the description/children wrapping will get thrown off.

I think we'd be best suited to avoid using a Spacer here to account for the icon. From that other comment chain, I think either:
- The entire Alert should be Pressable (we can move the chevron icon to be in the View row flex structure down here) or
- We should use hitSlop to pad out the pressable area regardless of icon presents (again we can move the chevron icon to be in the View row flex structure down here)

Reply:
Added `hitSlop` to expand coverage of the touchable area, but ran into a weird issue when moving the expand icon out of the Pressable. The icon itself wasn't responding to touches, but everywhere else within the hitSlop area was. Couldn't find too much about it on Google except that sometimes hitSlop doesn't work well with SVGs/icons, but strangely it works fine on the left Info icon. See video:


https://github.com/department-of-veterans-affairs/va-mobile-library/assets/786704/2226da49-be34-4360-955b-314af1a34a0f



Ended up just adding an invisible icon I've called `spacerIcon` which is guaranteed match the width of the expand icon at any scale. I tried using a Spacer instead and calculating the proper width with fontScale but it doesn't quite work since the icon and spacer scale at different rates due to the maxWidth.

Reply:
Weird--but also something I think I may have run into once upon a time way back in my prior job of touchable weirdness with SVGs.

Unfortunate, but I think the solution makes sense given the odd circumstances.